mitmproxy/tools/console/flowview.py

```diff
@@ -117,7 +117,13 @@ def tab_http_request(self):
     def tab_http_response(self):
         flow = self.flow
         assert isinstance(flow, http.HTTPFlow)
-        if self.flow.intercepted and flow.response:
+
+        # prevent renaming when websocket messages are intercepted
+        if (
+            self.flow.intercepted
+            and flow.response
+            and (not flow.websocket or len(flow.websocket.messages) == 0)
+        ):
```

Comment:
```suggestion
        # there is not good way to detect what part of the flow is intercepted,
        # so we apply some heuristics to see if it's the HTTP response.
        response_is_intercepted = (
            self.flow.intercepted
            and flow.response
            and not (flow.websocket and flow.websocket.messages)
        )
        if response_is_intercepted:
```

 - the comment you added is useful in the context of your change, but if you look at this code without context it's a bit out of place
 - adding a variable for the boolean expression is a neat trick to convey intent of what the expression is supposed to check.